### ISSUE TITLE
Emissions Module View: Long-Term Fuel Flow -#23

### ISSUE THREAD

**Issue by MaeganWood64** [US-EPA-CAMD/PI-1#4330]
Current Template Name: Long-Term Fuel Flow
New Name: Long Term Fuel Flow
Order in dropdown: #23

## Step #1:
Convert the following table in the SQL Server client database to Postgres.

**Table is in the ECMPS client SQL Server database**

**Table will need to be in the camdecmps and camdecmpswks schemas**
camdecmps.emission_view_
camdecmpswks.emission_view_

## Step #2:
Convert the following procedure in the SQL Server client database to Postgres.

**Procedure is in the ECMPS client SQL Server database**

**Procedure will only need to be in the camdecmpswks schemas**
camdecmpswks.refresh_emission_view_

## Step #3:
Load view definition into the camdaux.dataset, camdaux.datatable, & camdaux.datacolumn tables

**Run these on the ECMPS Sql Server client database to get view names & columns**
select * from [Emissions].[Grid]
select * from [Emissions].[GridColumn] where grid_id = <enter id> order by column_order

**This will extract columns into a sql statement that can be placed into load scripts...**
select '(datatableId, ' + cast(column_order as varchar)  + ', ''' + lower(column_key) + ''', ''' + column_name + '''),'
from [Emissions].[GridColumn] where grid_id = <enter id> order by column_order

**Comment by MaeganWood64** [comment]
Need Further Clarity on the curent table

**Comment by MaeganWood64** [comment]
On hold as we are still waiting on clarity for long-term fuel flow suppo rt
